pastas/modelstats.py

```diff
@@ -202,6 +202,33 @@ def nse(
         obs = self.ml.observations(tmin=tmin, tmax=tmax)
         return metrics.nse(obs=obs, res=res, weighted=weighted, **kwargs)
 
+    @model_tmin_tmax
+    def nnse(
+        self,
+        tmin: Optional[TimestampType] = None,
+        tmax: Optional[TimestampType] = None,
+        weighted: bool = False,
+        **kwargs,
+    ) -> float:
+        """Normalized Nash-Sutcliffe Efficiency for model fit .
+
+        Parameters
+        ----------
+        tmin: str or pandas.Timestamp, optional
+        tmax: str or pandas.Timestamp, optional
+        weighted: bool, optional
+            If weighted is True, the variances are computed using the time step
+            between observations as weights. Default is False.
+
```

Comment:
Any references we can add for this metric? I never heard of it so adding a reference would be helpful for users new to this metric :)

Reply:
Never mind, REF found below ;-)